sdk/core/azure-core/inc/azure/core/http/http.hpp

```diff
@@ -486,10 +510,10 @@ namespace Azure { namespace Core { namespace Http {
      */
     explicit Request(HttpMethod httpMethod, Url url, bool downloadViaStream)
         : Request(
-            httpMethod,
-            std::move(url),
-            NullBodyStream::GetNullBodyStream(),
-            downloadViaStream)
+              httpMethod,
+              std::move(url),
+              NullBodyStream::GetNullBodyStream(),
+              downloadViaStream)
```

Comment:
Revert this formatting change, not sure why we keep introducing it.

Reply:
I would like to understand what formatting/text editor you are using because we keep making this change, and if I format the `http.hpp` file again in VS, it reverts it:
![image](https://user-images.githubusercontent.com/6527137/102439678-8790df80-3fd3-11eb-9008-5535019ad7de.png)